content/en/docs/concepts/configuration/secret.md

```diff
@@ -717,6 +717,8 @@ A container using a Secret as a
 Secret updates.
 {{< /note >}}
 
+## Immutable Secrets {#secret-immutable}
```

Comment:
Hello @sftim .
I wonder if the `Immutable Secrets` section belongs between the "Using Secrets as files ..." and
"Using Secrets as environment variables ..." sections? The new heading seems
to break up the previous content.

Reply:
Following up with this:

Is this meant to be a `###` (i.e., the same nesting level as the two topics Karen mentions) instead?

Reply:
I've moved the section - how's this?

Reply:
@sftim lgtm